src/components/Contacts/ContactDetails/ContactDetailsTab/Other/ContactDetailsOther.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsTabQuery } from '../ContactDetailsTab.generated';
+
+const ContactOtherContainer = styled(Box)(({ theme }) => ({
+  margin: theme.spacing(1, 1, 1, 5),
+}));
+
+const ContactOtherTextContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  marginLeft: theme.spacing(2),
+}));
+
+const ContactOtherTextLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.hint,
+  marginRight: '5px',
+}));
+
+interface ContactDetailsOtherProp {
+  contact: ContactDetailsTabQuery;
```

Comment:
This would be better typed if you made a fragment for just the fields needed by this component and used that fragment in the parent query. Then you could just use the generated type for that fragment.

I think it's better for each component to specify the fields they need. But this is a small component so it could be ok 🤷‍♂️